cli/manifest/types/types.go

```diff
@@ -16,10 +17,12 @@ import (
 type ImageManifest struct {
 	Ref        *SerializableNamed
 	Descriptor ocispec.Descriptor
+	Raw        []byte `json:",omitempty"`
 
 	// SchemaV2Manifest is used for inspection
-	// TODO: Deprecate this and store manifest blobs
```

Comment:
LOL, so there even was a TODO for this (added in https://github.com/docker/cli/commit/1fd2d66df89890ca7d830a862d66e1275337ef65 / https://github.com/docker/cli/pull/1156)

Looks like there even was a PR that did so, but got closed? 🤔 https://github.com/docker/cli/pull/1144